orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/DeadMessageHandler.kt

```diff
@@ -17,16 +17,19 @@
 package com.netflix.spinnaker.orca.q.handler
 
 import com.netflix.spinnaker.orca.ExecutionStatus.TERMINAL
+import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
 import com.netflix.spinnaker.orca.q.*
 import org.slf4j.LoggerFactory
-import org.springframework.stereotype.Component
 
-@Component open class DeadMessageHandler {
+open class DeadMessageHandler(
```

Comment:
I don't think it was a good idea for this to be a `@Component` since it was overridden. I've made that explicit in the configuration class instead.